usb/cherryusb/host/host_cdc_msc_hid/main.c

```diff
@@ -0,0 +1,23 @@
+#include "pico/stdlib.h"
+#include "pico/time.h"
+#include "usbh_core.h"
+#include <stdio.h>
+#include <string.h>
+#include "FreeRTOS.h"
+#include "task.h"
+
+#include "usb_host.c"
+
+int main(void)
+{
+    stdio_init_all();
+    printf("CherryUSB host cdc msc example\n");
```

Comment:
Apologies for the stupid question, but what does this example actually do?

Reply:
Actually our class driver are all automatic loaded without call some apis, we include usb_host.c which has cdc + msc + hid transfer demo.

Reply:
In usb_host.c, when class enums done and go to class callback, then we will create a thread to do transaction, so in main.c we only use usbh_init api. For other class drivers are the same.